core/src/main/scala/org/allenai/common/FileUtils.scala

```diff
@@ -0,0 +1,73 @@
+package org.allenai.common
+
+import _root_.au.com.bytecode.opencsv.CSVReader
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import scala.io.{ BufferedSource, Codec, Source }
+
+/** Various convenient utilities for reading files and resources. */
+object FileUtils extends Logging {
+
+  /** Get a file as a buffered source. */
```

Comment:
Dropping this method and instead just using `Source.fromFile(file)` with implicit `codec` where this method was used.